tcnapi/exile/gate/v2/public.proto

```diff
@@ -299,6 +300,18 @@ service GateService {
       body: "*"
     };
   }
+  rpc Transfer(TransferRequest) returns (TransferResponse) {
+    option (google.api.http) = {
+      get: "/tcnapi/exile/gate/v2/transfer"
+      body: "*"
+    };
+  }
+  rpc CallerRequeue(CallerRequeueRequest) returns (CallerRequeueResponse) {
```

Comment:
This method must be named different or we should drop it completely

Reply:
fixed